### PR TITLE
overlay run with alpha animation

### PR DESCRIPTION
**Pull request by katsuya-yamada-eureka**
・now, CardStackView#swipe run when alphaAnimation is not work..
・but, this PullRequest will provide overload method of "swipe". made possible setting overlay run with AlphaAnimations.

### CODE REVIEW

File: sample/src/main/java/com/yuyakaido/android/cardstackview/sample/MainActivity.java [hunk 1]
```diff
@@ -208,9 +208,10 @@ public void swipeLeft() {
         }
 
         View target = cardStackView.getTopView();
+        View targetOverlay = cardStackView.getTopView().getOverlayContainer();
 
         ValueAnimator rotation = ObjectAnimator.ofPropertyValuesHolder(
-                target, PropertyValuesHolder.ofFloat("rotation", -10f));
+                target, PropertyValuesHolder.ofFloat("rotation", 10f));
```
Comment (yuyakaido) [inline sample/src/main/java/com/yuyakaido/android/cardstackview/sample/MainActivity.java]: Why did you change this code?

File: sample/src/main/java/com/yuyakaido/android/cardstackview/sample/MainActivity.java [hunk 2]
```diff
@@ -233,9 +239,10 @@ public void swipeRight() {
         }
 
         View target = cardStackView.getTopView();
+        View targetOverlay = cardStackView.getTopView().getOverlayContainer();
 
         ValueAnimator rotation = ObjectAnimator.ofPropertyValuesHolder(
-                target, PropertyValuesHolder.ofFloat("rotation", 10f));
+                target, PropertyValuesHolder.ofFloat("rotation", -10f));
```
Comment (yuyakaido) [inline sample/src/main/java/com/yuyakaido/android/cardstackview/sample/MainActivity.java]: Same as above

File: cardstackview/src/main/java/com/yuyakaido/android/cardstackview/CardStackView.java [hunk 3]
```diff
@@ -261,19 +261,19 @@ public void performSwipe(Point point, final Animator.AnimatorListener listener)
...
-            getTopView().setOverlayAlpha(1f);
+            getTopView().setOverlayAlpha(overlayAnimatorSet);
         } else if (direction == SwipeDirection.Bottom){
             getTopView().showBottomOverlay();
-            getTopView().setOverlayAlpha(1f);
+            getTopView().setOverlayAlpha(overlayAnimatorSet);
         } else if (direction == SwipeDirection.Top){
             getTopView().showTopOverlay();
-            getTopView().setOverlayAlpha(1f);
+            getTopView().setOverlayAlpha(overlayAnimatorSet);
```
Comment (yuyakaido) [inline cardstackview/src/main/java/com/yuyakaido/android/cardstackview/CardStackView.java]: if overlayAnimatorSet is null, overlay does not work.
if it is null, we should call `setOverlayAlpha(1f)` .

File: sample/src/main/java/com/yuyakaido/android/cardstackview/sample/MainActivity.java (hunk 1 above)
Comment (katsuya-yamada-eureka) [reply sample/src/main/java/com/yuyakaido/android/cardstackview/sample/MainActivity.java]: Im sorry.. do not need modify
so, It was returned to the original

File: sample/src/main/java/com/yuyakaido/android/cardstackview/sample/MainActivity.java (hunk 2 above)
Comment (katsuya-yamada-eureka) [reply sample/src/main/java/com/yuyakaido/android/cardstackview/sample/MainActivity.java]: yes. same as above.
It was returned to the original

File: cardstackview/src/main/java/com/yuyakaido/android/cardstackview/CardStackView.java (hunk 3 above)
Comment (katsuya-yamada-eureka) [reply cardstackview/src/main/java/com/yuyakaido/android/cardstackview/CardStackView.java]: sorry.. I missed that. X)
so modified. Please confirm!
